packages/web/src/pages/chat-page/components/ChatComposer.module.css

```diff
@@ -7,20 +7,50 @@
   padding: var(--unit-2) var(--unit-4);
 }
 
+.input {
+  padding: var(--unit-1) var(--unit-4);
+}
+
+.input textarea {
+  /* Compensate for the larger icon */
+  line-height: 200%;
```

Comment:
How does this look with multiline text?

Reply:
Good call, looks jenk. Will look for another solution